project/src/types/offer.ts

```diff
@@ -0,0 +1,22 @@
+type Offer = {
+  bedrooms: number,
+  cityName: string,
+  description: string,
+  goods: string[],
+  hostAvatarUrl: string,
+  hostId: number,
+  hostIsPro: boolean,
+  hostName: string,
+  id: number,
+  images: string[],
+  isFavorite: boolean,
+  isPremium: boolean,
+  maxAdults: number,
+  previewImage: string,
+  price: number,
+  rating: number,
+  title: string,
+  type: string,
```

Comment:
тут должен быть енам
```suggestion
  type: OfferType,
```